packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -32,89 +32,100 @@
+                    <oui-tile-button
+                        data-on-click="$ctrl.startRescueInstance()"
+                        data-ng-if="$ctrl.instance.canBeHardRebooted()"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_rescue_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-on-click="$ctrl.endRescueInstance()"
+                        data-ng-if="$ctrl.instance.status === 'RESCUE'"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_rescueEnd_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-on-click="$ctrl.softRebootInstance()"
+                        data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE'"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_softReboot_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-on-click="$ctrl.hardRebootInstance()"
+                        data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE'"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_hardReboot_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-on-click="$ctrl.reinstallInstance()"
+                        data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE'"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_reinstall_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-on-click="$ctrl.resumeInstance()"
+                        data-ng-if="$ctrl.instance.statusGroup === 'SUSPENDED'"
+                    >
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_resume_label"
+                        ></span>
+                    </oui-tile-button>
+                    <oui-tile-button data-on-click="$ctrl.deleteInstance()">
+                        <span
+                            data-translate="pci_projects_project_instances_instance_action_delete_label"
+                        ></span>
+                    </oui-tile-button>
+                </oui-tile>
+            </div>
+
+            <!-- Advices -->
+            <div class="mt-4">
+                <ovh-advices
+                    service-type="instance"
```

Comment:
This could be added as a constant instead.

Reply:
I think it's ok because its only referred in one place